apps/server/src/modules/authorization/api/dto/authorization.reponse.ts

```diff
@@ -0,0 +1,45 @@
+import { Permission } from '@shared/domain/interface';
+import { ApiProperty } from '@nestjs/swagger';
+import { Action, AuthorizableReferenceType, AuthorizationContext } from '../../domain';
+
+export class AuthorizedReponse implements AuthorizationContext {
+	@ApiProperty({
+		enum: Permission,
+		isArray: true,
+		description: 'Needed user permissions based on user role, that are needed to execute the operation.',
+		example: [Permission.ACCOUNT_VIEW, Permission.BASE_VIEW],
+	})
+	requiredPermissions: Permission[];
```

Comment:
Why return all the inputs and not just the result?

Reply:
Das isAuthorized ohne was überhaupt authorisiert hat dann relativ wenig Aussagekraft. 
Der Anfragende muss nicht unbedingt auch derjenige sein der die Antwort erhält.

Reply:
"Der Anfragende muss nicht unbedingt auch derjenige sein der die Antwort erhält." Das versteh ich nicht. Wie kann denn jemand anders die Antwort erhalten?

Reply:
done